### ISSUE TITLE
Error in `options` howto doc

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7791]
|<img alt="bob_f's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| bob_f reported|
|-|-|
|Trac ID|trac#7791|
|Type|enhancement|
|Created|2015-02-15 17:32:31Z|
|Branch|https://github.com/twisted/twisted/tree/docs-howto-options-7791|

http://twistedmatrix.com/documents/current/core/howto/options.html

Should say "usage.Options" but says "BaseOptions" - see attached patch.

Attachments:

* [options.html.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/43d/43dd3a9d0da0bf0ff2e627fe41c1f5d3c2726129/82e0f93e2b40fb0f26a257a15079f0d2b2846041.patch) (319 bytes) - added by bob_f on 2015-02-15 17:33:09Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7791 7791
type__enhancement enhancement
reporter__bob_f bob_f
priority__normal normal
milestone__ 
branch__branches_docs_howto_options_7791 branches/docs-howto-options-7791
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1424021551055006 1424021551055006
changetime__1447940646025777 1447940646025777
version__None None
owner__bob_f bob_f
cc__oubiwann
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to bob_f|
|-|-|

Many thanks for your changes.

I have no idea why this review is listed only now in the review queue.

I have adapted your patch to latest docs.

Waiting for test and if all is ok will merge.

Again, many thanks for your contribution!

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra commented|
|-|-|

Replying to [adiroiban](#note_3):
> Many thanks for your changes.
> 
> I have no idea why this review is listed only now in the review queue.
> 
It's because I added the review keyword, it was never put up for review before.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_6"></a>|@adiroiban set status to `closed`|
|-|-|

(In [46288]) Merge docs-howto-options-7791: Fix error in options howto doc.

Author: bob_f
Reviewer: adiroiban, thijs
Fixes: [#7791](https://github.com/twisted/twisted/issues/7791)
